_data/whitelist.yml

```diff
@@ -7,8 +7,12 @@ projects:
   goal: Amicable Numbers is an independent research project that uses Internet-connected
     computers to find new amicable pairs. Currently searching the 10^20 range.
   sponsor: Private
-  cpu: 'yes'
-  gpu: 'yes'
+  supports_windows: 'y'
```

Comment:
why are some of these single quotes and others double quotes?

Reply:
honestly, out of laziness, double quotes would be preferred right?

Reply:
Double quotes are what the template prescribes, so yes